public/stylesheets/landing.css

```diff
@@ -0,0 +1,43 @@
+canvas {
```

Comment:
please do not style elements directly unless absolutely necessary. Give this a class name.